stan/math/rev/mat/functor/adj_jac_apply.hpp

```diff
@@ -48,6 +48,106 @@ template <class F, class Tuple>
 constexpr auto apply(const F& f, const Tuple& t) {
   return apply_impl(f, t, std::make_index_sequence<std::tuple_size<Tuple>{}>{});
 }
+
+/**
+ * build_y_adj takes the adjoint from the vari pointed to
+ * by y_vi_[0] and stores it in y_adj
+ *
+ * @tparam size dimensionality of M_
+ * @param y_vi_ pointer to pointer to vari
+ * @param M_ ignored in this specialization
+ * @param[out] y_adj reference to variable where adjoint is to be stored
```

Comment:
If you're going to list in/out, you should do them all.  You don't need to say "in this specialization" either---it's implied that you're documenting this specialization.